fabric-input-events-v1/src/main/java/net/fabricmc/fabric/api/client/input/v1/FabricMouse.java

```diff
@@ -16,29 +16,47 @@
 
 package net.fabricmc.fabric.api.client.input.v1;
 
+import net.fabricmc.fabric.impl.client.input.FabricKeyboardImpl;
 import net.fabricmc.fabric.impl.client.input.FabricMouseImpl;
 
 public final class FabricMouse {
 	private FabricMouse() {
 	}
 
+	/**
+	 * Get the current X position of the mouse.
+	 */
 	public static double getX() {
 		return FabricMouseImpl.getX();
 	}
 
+	/**
+	 * Get the current Y position of the mouse.
+	 */
 	public static double getY() {
 		return FabricMouseImpl.getY();
 	}
 
+	/**
+	 * Get a bitmask of the mouse buttons currently being pressed.
+	 */
 	public static int getPressedButtons() {
 		return FabricMouseImpl.getPressedButtons();
 	}
 
+	/**
+	 * Check if the given mosue button is currently being pressed.
+	 *
+	 * @param button the GLFW.GLFW_MOUSE_BUTTON to check
+	 */
 	public static boolean isButtonPressed(int button) {
 		return FabricMouseImpl.isButtonPressed(button);
 	}
 
+	/**
+	 * Get the GLFW.GLFW_MOD modifier keys currently being pressed.
+	 */
 	public static int getModKeys() {
```

Comment:
This seems duplicated from keyboard stuff?

Reply:
Already resolved in 
29ab4bc?

It's added to `FabricMouse` as well because that's the way GLFW does it. Seems useful when you want to implement e.g. CTRL + LMB

Reply:
OS events typically duplicate modifier keys in mouse events as well.